subprojects/build-cache/src/main/java/org/gradle/caching/internal/controller/service/BuildCacheServiceHandle.java

```diff
@@ -22,6 +22,9 @@
 
 import javax.annotation.Nullable;
 import java.io.Closeable;
+import java.io.File;
+import java.util.Optional;
+import java.util.function.Function;
 
 public interface BuildCacheServiceHandle extends Closeable {
```

Comment:
Let's rename this to `RemoteBuildCacheServiceHandle`.

Reply:
done